src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -354,7 +354,7 @@ def create(
             rbd_image_name: str,
             rbd_pool: str = "rbd",
             create_image: Optional[bool] = True,
-            size: Optional[int] = 1024,
+            rbd_image_size: Optional[int] = 1024,
```

Comment:
i just realized that there are more than one consumer for the REST API now that we know (dashboard and vSphere plugin), and for them this is going to be a breaking change unless they are not notified. Was this a required change or should have been avoided? If we are moving ahead with the change, probably its worth letting them know since there isn't an native backward compatibility in the API.